vms/platformvm/block/builder/builder_test.go

```diff
@@ -438,10 +439,10 @@ func TestBuildBlockInvalidStakingDurations(t *testing.T) {
 		uint64(now.Unix()),
 		uint64(validator2EndTime.Unix()),
 		ids.GenerateTestNodeID(),
-		preFundedKeys[2].PublicKey().Address(),
+		genesistest.Keys[2].PublicKey().Address(),
```

Comment:
Can we keep the `preFundedKeys` cadence? We use `secp256k1.TestKeys()` and I don't think that's going to change